README.md

```diff
@@ -3,3 +3,37 @@
 [![Build Status](https://travis-ci.org/ivotron/slurmp.svg?branch=master)](https://travis-ci.org/ivotron/slurmp)
 
 Utilities for running parameter sweeps by providing a Jinja2 template and a parameter space (in YAML).
+
+
+## Installation
+```bash
+git clone https://github.com/ivotron/sweepj2 /tmp/sweep2
+pip install /tmp/sweepj2/bin
+```
```

Comment:
we will upload the package to pypi.org, so this will be `pip install sweepj2`

Reply:
I have updated this.